.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Fixes #issuenumber ; refs #issuenumber
+
+Present tense short summary (50 characters or less)
+
+More detailed description, if necessary. Try to be as descriptive as you can: even if you think that the PR content is obvious, it may not be obvious to others. Include tracebacks if helpful, and be sure to call out any bits of the PR that may be work-in-progress.
+
+Description can have multiple paragraphs and you can use code examples inside:
+
+``` ruby
+class PostsController
+  def index
+    respond_with Post.limit(10)
+  end
+end
+```
+
+Changes proposed in this pull request:
+* 
+* 
+* 
+
+@projecthydra-labs/hyrax-code-reviewers
```

Comment:
We might want to use a different group here. Perhaps @projecthydra/hydra-contributors  ?